test/remote/push_tests.js

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
The tests in this "remote" directory all use the client API to hit a more-or-less live server API instance.  I like the existence of this test but I wonder if it's out of place in this directory.